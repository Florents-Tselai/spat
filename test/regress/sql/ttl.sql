-- TTL

SELECT SPSET('expkey1', 'expvalue1', ttl=> '1 second');
SELECT TTL('expkey1') < '1 second' ;
