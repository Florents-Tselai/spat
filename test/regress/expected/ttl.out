-- TTL
SELECT SPSET('expkey1', 'expvalue1', ttl=> '1 second');
   spset   
-----------
 expvalue1
(1 row)

SELECT TTL('expkey1') < '1 second' ;
 ?column? 
----------
 t
(1 row)

